config/pmd-test.xml

```diff
@@ -32,7 +32,7 @@
             file to ease navigation and search for test-->
         <exclude name="ExcessiveImports"/>
         <!--test should be named as their main class-->
-        <exclude name="AbstractNaming"/>
+        <exclude name="ClassNamingConventions"/>
```

Comment:
This could actually be removed. No errors show up with it removed.
This check doesn't prevent naming non-abstract classes with abstracting naming convention.